recipes/pipes/all/conanfile.py

```diff
@@ -0,0 +1,51 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class PipesConan(ConanFile):
+    name = "pipes"
+    description = "Pipelines for expressive code on collections in C++"
+    license = "MIT"
+    topics = ("pipes", "functional-programming")
+    homepage = "https://github.com/joboccara/pipes"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "compiler"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 14
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "15",
+            "gcc": "5",
+            "clang": "3.4",
+            "apple-clang": "5.1",
+        }
+
+    def configure(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, self._minimum_cpp_standard)
+        min_version = self._minimum_compilers_version.get(str(self.settings.compiler))
+        if not min_version:
+            self.output.warn(f"{self.name} recipe lacks information about the {self.settings.compiler} compiler support.")
```

Comment:
Format strings might cause a problem, since conan supports Python 3.5. Have to change that.